.github/workflows/CI.yaml

```diff
@@ -66,19 +82,39 @@ jobs:
         with:
           name: JaCoCoResultsUnitTests
           path: ./dataland-backend/build/jacoco/*.exec
+
+  unit-tests-frontend:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'workflow_dispatch'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 17
+          cache: gradle
+      - name: Execute Frontend Unit Tests
+        run: ./gradlew :dataland-frontend:npm_run_testunit
```

Comment:
Wollen wir das - aus Konsistenzgründen - nicht auch im Cypressbase-Container ausführen?

Reply:
@SiegfriedSobkowiak  und ich hatten die Frontend Unit Tests mal in einem Temurin-Base-Image und in einem weiteren Versuch in einem Ubuntu-Base-Image laufen lassen, was zu Errors geführt hat.
Als Quick Fix haben wir die Frontend-Unit-Tests dann erstmal im Runner belassen, weil es dort funktioniert hat (quasi wie vorher).

Also Zeitinvestitionsfrage, ob man das nicht versucht doch noch im Container laufen zu lassen.

Reply:
pass so.